teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(name, age, specialization)
+    super(name, age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] It would be nice to re-arrange your super method here to match how the Person class is initialized.
- [ ] You are missing the `parent_permission` parameter. Kindly add it.

Reply:
Thanks a lot, @TedLivist  I fixed it! 🤓  🧑‍💻